### DISCUSSION TITLE
Rejected for distance learning

### DISCUSSION THREAD

**Discussion by lancewilhelm** [community/community#50458] [GitHub Education]
I keep receiving the following rejection:
Your application cannot be reviewed until you fix the following:
You appear not to be near any campus location for the school you have selected. If you are a distance learner then your school-provided academic affiliation documentation must state so.

However, my uploaded document explicitly states that my campus is "online" (student I.D. masked for this post). Can someone help reconcile this error?

![Student Information2](https://user-images.githubusercontent.com/11323174/226217549-432d77e1-fd50-4c99-9ddb-0e8bd85b8310.jpg)

**Comment by jsuddsjr** [comment]
Someone needs to retrain the AI to understand the meaning of "Campus: Online." I'm having similar issues with "Subprogram: Online." I'm not sure how to make it more obvious.
